src/hooks/web3/useBridgeSommToEthTx.tsx

```diff
@@ -11,94 +10,34 @@ import { analytics } from "utils/analytics"
 import { useAccount, useSigners } from "graz"
 import { txClient } from "src/vendor/ignite/gravity.v1"
 import { ethers } from "ethers"
+import { useImportToken } from "hooks/web3/useImportToken"
 
 export const useBridgeSommToEthTx = () => {
-  const { CONTRACT } = config
-  // Currently `close` have a bug it only closes the last toast appeared
-  // TODO: Fix `close` and implement it here https://github.com/strangelove-ventures/sommelier/issues/431
   const { addToast, update, closeAll } = useBrandedToast()
   const [isLoading, setIsLoading] = useState(false)
 
   const { data } = useAccount()
   const { signerAmino } = useSigners()
 
-  const TxHashToastBody = ({
```

Comment:
I feel like it was better to leave this & BridgeTxHashToastBody as reusable constants